reference-architecture/vmware-ansible/docker/vsphere_guest.py

```diff
@@ -0,0 +1,1891 @@
+#!/usr/bin/python
```

Comment:
It is. Unfortunately, it lacked the functionality to add annotations via a VM clone. I extended that functionality with code updates and submitted a PR in July. It has not been merged yet.